sniffles/ruletrafficgenerator.py

```diff
@@ -985,13 +985,14 @@ def updateSequence(self, dir="to server", data_len=1):
                 self.current_seq_b_to_a += data_len
                 self.current_ack_a_to_b = self.current_seq_b_to_a
 
-# !!Create background traffic.
+
 class BackgroundTraffic(TrafficStream):
     def __init__(self, rule=None, sconf=None, start_sec=-1, start_usec=0):
         super().__init__(None, sconf)
         self.proto = rule.getProto()
         self.rule = rule
         self.rand = False
+        self.full_match = True
```

Comment:
I've added full_match variable to stay on because I felt that partial matching (i.e. change of last 1~2 character in the packet) is unnecessary for background traffics.